harness/determined/cli/task.py

```diff
@@ -38,19 +38,19 @@ def agent_info(t: Dict[str, Any]) -> Union[str, List[str]]:
     ]
     values = [
         [
-            task["task_id"],
```

Comment:
ditto. if you think this might add some risk feel free to ignore it.

Reply:
Do you mind elaborate? I am not sure the connection between `make -C proto/ clean fmt build && make -C bindings clean build` and `task["task_id"]`. I change it to `task["taskId"]` because this is the default style from the gRPC gateway response.

Reply:
I was suggesting since you're changing this we could probably update this code to use our typed bindings `def get_GetTasks(` so that we're not blindly reaching into an untyped `task` 
that'd probably involve changing https://github.com/determined-ai/determined/blob/554764f61a6ba135f8f6b6c343782dbed9cf0415/harness/determined/cli/task.py#L62 to use `bindings.get_GetTasks`

but feel free to leave it for another day

Reply:
Thanks, get it! Will use `v1GetTaskResponse` object in `bindings`. It seems better than an untyped `task`.